bin/uscis

```diff
@@ -24,6 +24,8 @@ Promise.all(cases.map((c) => caseStatus(c)))
       }
     })
 
+  }, (err) => {
+    console.log(err.message.red)
```

Comment:
@marlonpp I might be wrong. For example

``` js
promise.then(resolve, reject)
```

```
promise.then(resolve).catch(reject)
```

If we reject the promise, it will go to `catch` clause, right?

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch

Thanks again.